admin_guide/managing_networking.adoc

```diff
@@ -1470,6 +1470,40 @@ to accept any connection allowed by each policy. That is,  connections on any
 port from pods in the *_same_* namespace, and connections on ports `80` and
 `443` from pods in *_any_* namespace.
 
+[[admin-guide-networking-networkpolicy-namespace-reduce-ovs-flow-rules]]
+=== Using Namespaces to reduce OVS Flow Rules generated by NetworkPolicy
+
+`NetworkPolicy` objects allow you to isolate pods that are differentiated from
+one another by labels, within a namespace.
+
+It is inefficient to apply `NetworkPolicy` objects
+to large numbers of individual pods in a single namespace.
+Pod labels do not exist at the IP level, so `NetworkPolicy` objects generate
+a separate OVS flow rule for every single possible link between every pod
+selected with `podSelector`.
+
+If the `_spec_` `podSelector` and
+the `_ingress_` `podSelector` within a `NetworkPolicy` object each match 200
+pods, then 40000 (200*200) OVS flow rules are generated. This causes the nodes
+to become NotReady, which leads to a
+"KubeletNotReady PLEG is not healthy" error.
```

Comment:
I wouldn't get into the specifics of that particular bug. The exact symptoms may change over time (especially if do some targeted bugfixes for that customer). Just say that it's inefficient and may slow down the machine.

Reply:
@danwinship 
I'll omit the specific bug, but do you think it's useful to keep the figures in?